api_app/api/routes/api.py

```diff
@@ -157,5 +158,20 @@ async def get_workspace_swagger(workspace_id, request: Request, workspace_repo=D
 
     return swagger_ui_html
 
-workspace_router.include_router(workspace_swagger_router)
+
+swagger_disabled_router = APIRouter()
+
+
+@swagger_disabled_router.get("/docs", include_in_schema=False, name="swagger_disabled")
```

Comment:
Swagger is also active in the workspace scope, i.e. /api/workspaces/some_guid/docs. 
Does this handle it as well?